python-threatexchange/threatexchange/content_type/file_content.py

```diff
@@ -0,0 +1,30 @@
+import typing as t
+from content_base import ContentType
+from photo import PhotoContent
+from video import VideoContent
+
+class FileContent(ContentType):
+    """
+    Content representing a file. Determines if a file is a photo or video based on file extension.
+    """
+
+    VALID_PHOTO_EXTENSIONS = {".jpg", ".jpeg", ".png", ".gif"}
+    VALID_VIDEO_EXTENSIONS = {".mp4", ".avi", ".mov", ".mkv"}
+
+    def __init__(self, file_name: str):
+        self.file_name = file_name
```

Comment:
blocking q: Currently, I don't any ContentType has an implementation for instantiation (we couldn't agree on what it should mean at the time, and so just saved it for the future). What would be your arguments to make file the first one?